src/components/edit/Triggers.tsx

```diff
@@ -1,16 +1,24 @@
 'use client'
 import { useState, useCallback } from 'react'
+import { GraphQLError } from 'graphql'
 import { Trash } from '@phosphor-icons/react/dist/ssr'
 import { PlusIcon, PlusCircleIcon } from '@heroicons/react/20/solid'
 import { MobileDialog, DialogContent, DialogTrigger } from '../ui/MobileDialog'
-import DeleteAreaForm, { DeleteAreaProps } from './DeleteAreaForm'
+import DeleteAreaAndClimbForm from './DeleteAreaAndClimbForm'
 import AddAreaForm, { AddAreaFormProps } from './AddChildAreaForm'
 import { toast } from 'react-toastify'
 import Tooltip from '../ui/Tooltip'
 
-export type DeleteAreaTriggerProps = DeleteAreaProps & {
+interface DeleteAreaTriggerProps {
```

Comment:
Agreed, since we're manually delegating props down (as opposed to splatting them down), I think no reason to try and share exact props.